### ISSUE TITLE
Show when credentials where updated last

### ISSUE THREAD

**Issue by vlerenc** [gardener/dashboard#945]
**What would you like to be added**:
This is not strictly a Dashboard topic, rather a generic one, but it would be made visible explicitly there (cc @ThormaehlenFred): Can we please track when credentials were updated last (when the credentials within the secret were actually modified). This would require something like a mutating web hook, because end users could also update their secrets directly via `kubectl`. When we have it, we could show it in the Dashboard. It wouldn't show how old the credentials really are (in the infrastructure), because we cannot check that, but it would give an indication to end users nonetheless, which is currently missing.

**Why is this needed**:
To show end users (approx.) when they updated their infrastructure credentials last. This is required for transparency reasons (show age) and to remind people (automatically) to rotate their infrastructure credentials (regularly).

/area security

**Comment by petersutter** [comment]
> This is not strictly a Dashboard topic, rather a generic one, but it would be made visible explicitly there

true, we can certainly display when it was last updated, however would such a mutating webhook be part of the gardener? If yes we should also create a corresponding issue there as prerequisite, otherwise I feel this will never be picked up

**Comment by vlerenc** [comment]
Less relevant now that we have `WorkloadIdentity` which is the safer and preferred way how to authorise Gardener to manage the user's cloud provider resources.

**Comment by grolu** [comment]
/close
as discussed
